platform/client/src/forms/volunteerAction.ts

```diff
@@ -15,21 +15,14 @@ const volunteerActionForm = new SimpleSchema({
         uniforms: {
             readonly: true
         }
-    },
+    }/*,
```

Comment:
`products` are commented for now as we'll need to fetch them separately - should be a concatenation of `products[i].product.label` I think after the addition of product model. - see https://github.com/aerogear/OpenVolunteerPlatform/pull/56